Tools/Scripts/webkitdirs.pm

```diff
@@ -3129,8 +3145,8 @@ sub findOrCreateSimulatorForIOSDevice($)
         $simulatorName = "iPad Pro " . $simulatorNameSuffix;
         $simulatorDeviceType = "com.apple.CoreSimulator.SimDeviceType.iPad-Pro--9-7-inch-";
     } else {
-        $simulatorName = "iPhone SE " . $simulatorNameSuffix;
-        $simulatorDeviceType = "com.apple.CoreSimulator.SimDeviceType.iPhone-SE";
+        $simulatorName = "iPhone 12 " . $simulatorNameSuffix;
+        $simulatorDeviceType = "com.apple.CoreSimulator.SimDeviceType.iPhone-12";
```

Comment:
iPhone SE (3rd Generation) would probably be good, too, but I don't object to iPhone-12 being the target.